tools/c7n_azure/tests_azure/tests_resources/test_app_service_plan.py

```diff
@@ -188,21 +188,25 @@ def test_resize_consumption_linux(self, logger):
             'name': 'test-azure-appserviceplan-linux',
             'resource': 'azure.appserviceplan',
             'filters': [
+                {'resourceGroup': 'test_appserviceplan-linux'},
                 {'type': 'value',
                  'key': 'name',
-                 'op': 'eq',
+                 'op': 'ne',
                  'value_type': 'normalize',
-                 'value': 'cctest-consumption-linux'}
+                 'value': 'cctest-appserviceplan-linux'}
```

Comment:
I think there actually is a predictable name for the consumption plan (Ex: `WestUSLinuxDynamicPlan`)

Reply:
Yeah, however I wanted to avoid hardcoding location in multiple different places